src/gauges/bartio/defaultGaugeList.json

```diff
@@ -155,6 +155,18 @@
         "0x23bc120feA570576E6D7bAe5195c9D7CD3bB6615",
         "0x0E4aaF1351de4c0264C5c7056Ef3777b41BD8e03"
       ]
+    },
+    {
+      "beraRewardsVault": "0x76f36Cc45478F3f84574937976168af0cbD44F02",
+      "lpTokenAddress": "0x760a5Cb9C84C16264A819A43C6c57F8dcE13662d",
+      "mintUrl": "https://app.kodiak.finance/#/liquidity/islands/0x760a5cb9c84c16264a819a43c6c57f8dce13662d?chain=berachain_bartio",
+      "name": "BUDS-BERA",
```

Comment:
`0x7507c1dc16935B82698e4C63f2746A2fCf994dF8` is WBERA. Please either change the name to match the token or change the token to match the name.

Reply:
Yeah, my bad. Changed it to WBERA.